arangod/V8Server/v8-vocindex.cpp

```diff
@@ -152,13 +152,13 @@ static void JS_DropIndexVocbaseCol(
   VPackBuilder builder;
   TRI_V8ToVPack(isolate, builder, args[0], false, false);
 
-  auto res = methods::Indexes::drop(collection, builder.slice());
+  auto res = methods::Indexes::drop(*collection, builder.slice());
 
-  if (res.ok()) {
-    TRI_V8_RETURN_TRUE();
+  if (res.fail()) {
+    TRI_V8_THROW_EXCEPTION(res);
   }
 
-  TRI_V8_RETURN_FALSE();
+  TRI_V8_RETURN_TRUE();
```

Comment:
The PR changes the behavior of `db.<collection>.dropIndex()` in case of non-existing indexes:

https://github.com/arangodb/arangodb/pull/18502/files#diff-ecec88c33adb7591ee6aa88e29b62ad52ef443611cba5e0f0ecac9b5725afdbaR4-R18

Reply:
I see, but question was about other stuff, why should we return bool now?
Is it useful? Or is it necessary for some backwards compatibility?